deepreg/registry.py

```diff
@@ -96,7 +96,15 @@ def build_from_config(self, category: str, config: dict, default_args=None):
 
         name = args.pop("name")
         cls = self.get(category=category, key=name)
-        return cls(**args)
+        try:
+            return cls(**args)
+        except TypeError as err:
+            raise ValueError(
+                f"Configuration  is not compatible for Class {cls} of category {category}.\n"
```

Comment:
Last comment:
i think it won't hurt to explicitly mention that "it is possible that an old format of configuration file may be being used."

and why two spaces between "Configuration" and "is"?

Reply:
done